adapters/beachfront/beachfront.go

```diff
@@ -218,40 +216,82 @@ func preprocess(request *openrtb.BidRequest) (beachfrontReqs beachfrontRequests,
 	}
 
 	if len(bannerImps) > 0 {
-		request.Imp = bannerImps
+		// request.Imp = bannerImps		// ... why was I doing this????
```

Comment:
It looks like you were doing this to feed just the banner imps to `getBannerRequest(request)`